src/NLog/Targets/Wrappers/AsyncRequestQueue-T.cs

```diff
@@ -185,5 +197,38 @@ public void Clear()
                 _logEventInfoQueue.Clear();
             }
         }
+
+        /// <summary>
+        /// Raise event when queued element was dropped because of queue overflow
+        /// </summary>
+        /// <param name="logEventInfo">Dropped queue item</param>
+        private void OnLogEventDropped(AsyncLogEventInfo logEventInfo)
+        {
+            var logEventDropped = LogEventDropped;
+            if (logEventDropped == null)
+            {
+                return;
+            }
+
+            LogEventDroppedEventArgs eventArg = new LogEventDroppedEventArgs(logEventInfo);
+            logEventDropped.Invoke(this, eventArg);
```

Comment:
I prefer ` LogEventDropped?.Invoke(this, new LogEventDroppedEventArgs(logEventInfo));`

the same for `OnLogEventQueueGrows` 

less is more :)